### PR TITLE
Add Modal component to index entry file.

### PR DESCRIPTION
**Pull request by shadowglen**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Add Modal component to index entry file.

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by shadowglen** [comment]
@dzearing / @cliffkoh could you please help to review?
We load the entire Fabric as bundle, but the Modal component isn't exported.

File: packages/office-ui-fabric-react/src/Modal.ts [hunk 1]
```diff
@@ -1,2 +1 @@
 export * from './components/Modal/index';
-export { Modal as default } from './components/Modal/index';
```
Comment (cliffkoh) [inline packages/office-ui-fabric-react/src/Modal.ts]: why remove this line? This would be a breaking change for people already using it..

**Comment by shadowglen** [comment]
@dzearing is added to the review. #Closed

File: packages/office-ui-fabric-react/src/Modal.ts (hunk 1 above)
Comment (shadowglen) [reply packages/office-ui-fabric-react/src/Modal.ts]: Reverted thx

---
In reply to: [207694831](https://github.com/OfficeDev/office-ui-fabric-react/pull/5802#discussion_r207694831) [](ancestors = 207694831)

**Pull request merged**